.changeset/eleven-rings-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@digdir/designsystemet-react": major
```

Comment:
Was this intended to be `major`? Not sure how this will affect changeset versioning in prerelease mode 🤔

Reply:
Yes, we have done major every time there has been a breaking change already?

Reply:
We actually haven't, since everything has been in development and not released to production. A major changset would for example bump the v1.0.0 to v2.0.0 if we were releasing outside of prelease mode. 

I am however curious to see what changeset does now and we should get in the habit of thinking proper about sem ver so lets test and see what happens :)